### PR TITLE
feat(Checkbox, RadioButton): Baseline alignment by default

### PR DESCRIPTION
**Pull request by sergivillar**

### CODE REVIEW

File: src/checkbox.tsx [hunk 1]
```diff
@@ -106,6 +106,17 @@ const useStyles = createUseStyles(() => ({
         opacity: 0.5,
```
Comment (pladaria) [inline src/checkbox.tsx:106]: This screenshot is failing (note the checked vs unchecked states), do you know why?

**Comment by pladaria** [comment]
Results of running: `yarn preint-mistica-branch sfernandez/align-checkbox-radio-to-top`

https://jenkins.tuenti.io/job/CI-webapp-master-preintegration/2077/

**Comment by yceballost** [comment]
I added as a reviewer too to this PR @sergivillar

**Review by yceballost — APPROVED** [review]

File: src/radio-button.tsx [hunk 2]
```diff
@@ -144,8 +144,11 @@ const RadioButton: React.FC<PropsRender | PropsChildren> = ({value, id, dataAttr
             {rest.render ? (
                 rest.render(radio, disabled)
             ) : (
-                <Inline space={16} alignItems="center">
-                    {radio}
+                <Inline space={16}>
```
Comment (yceballost) [inline src/radio-button.tsx:147]: There is a little extra pixel in radio button that make the text is displaced to the right?
image

https://mistica-web-oo1gboweb-tuentisre.vercel.app/playroom#?code=N4Igxg9gJgpiBcIA8AhCAPABABwIZSgEsA7AcwF5gBGANgF8A%2BAHWM0yQGUAXXMAa0wBnPGBiUATABZGLNkgDCACxj8ARhkzFcAWzFNwytRn3NWbTABkIAJxjbMhbIICu9qBAA2NoYS6YdMFwANJiQxIIqXIHO1v5E2ISCYCSkmDAeviERUJjuaYTOgtrQsuZR2tjeJMlEUM7Efs5%2BHrjqtml%2B7l7t2rikWv4ZAI7OuAB0mACqfjDEhPb4mNok86VsAG6zhDohI4maEIJc1s45MOgw1sk8XIQQrM4eLdqQmC1t%2B3OChJhNg4R7bBpLAwXBrUIQbTFPJhCIjXBcMalJAAeiUKj46nQpjkACV8HcAOLWCDObDgrS6cj6ABWzkIYAAngBaABmJ18%2BnBuGs22Z73SHhgUFUjOpIAFHi5ZjYsFZuEeXAAargPM49CBVLgtFppWwceZODx%2BEIRGJqPQDeZ2PiiBAUE0uPdMOtVerxVqdWCQFbrZYbHYHE5XLlPN5vn4AsEIeFItFYgSEkkUmkMtHsqHwdaYAUitBMOVKrFqoRavVGs1WgGOqHujAln0BqqAaMJtM0nMFlAs%2BZlp2XVsdpg9oIDkcTmcLldfAi7g8njpXu8bJ9Ej8mj22M3AcC0rgIVDoBAY3DRoie6jbXcHVwncRfajuLw%2BAbLwSIMTSdgHyin-xTKiaDYiwIB0EAA

**Review by yceballost — CHANGES_REQUESTED** [review]

File: src/radio-button.tsx (hunk 2 above)
Comment (sergivillar) [reply src/radio-button.tsx:147]: checkbox size is 18px width, radio button is 20px width, this was this way before these changes, but I could change this, what is the correct value, 18 or 20?

Comment (yceballost) [reply src/radio-button.tsx:147]: both is correct but in design we have both of thems inside a box with size 24x24

Comment (sergivillar) [reply src/radio-button.tsx:147]: understood, @pladaria  @atabel did you have these kind of issue in the past? I could add a wrapper to the icons of 24px width, WDYT?

Comment (pladaria) [reply src/radio-button.tsx:147]: image
What @yceballost comments is not documented in radio button/checkbox specs

If you want to align the text simply put a smaller spacing (between the radio and the text) in radio buttons

File: src/checkbox.tsx [hunk 3]
```diff
@@ -163,8 +163,11 @@ const Checkbox: React.FC<RenderProps | ChildrenProps> = (props) => {
             {props.render ? (
                 props.render(iconCheckbox, labelId)
             ) : (
-                <Inline space={16} alignItems="center">
-                    {iconCheckbox}
+                <Inline space={16}>
+                    {/* Text3 wrapper added to have the same line-height and center checkbox with text */}
+                    <Text3 regular as="div">
+                        <div style={{position: 'relative', top: -2}}>{iconCheckbox}</div>
```
Comment (atabel) [inline src/checkbox.tsx:169]: can you explain why you need this style?

File: src/list.tsx [hunk 4]
```diff
@@ -478,7 +478,12 @@ const RowContent = (props: RowContentProps) => {
                     onPress={toggle}
                     dataAttributes={dataAttributes}
                 >
-                    <Control name={name} checked={isChecked} aria-labelledby={titleId} />
+                    <Control
+                        name={name}
+                        checked={isChecked}
+                        aria-labelledby={titleId}
+                        render={(check) => check}
```
Comment (atabel) [inline src/list.tsx:485]: why is this needed?

File: src/radio-button.tsx (hunk 2 above)
Comment (yceballost) [reply src/radio-button.tsx:147]: It's true, is not specified, but you can select any control in spec and see this bounding box. So we can remove it from design or add it to code. 

why a 24px wrapper could break implementations?

File: src/checkbox.tsx (hunk 3 above)
Comment (sergivillar) [reply src/checkbox.tsx:169]: yep, this is to have a perfect pixel centered radio/checkbox with the first line of text (@pladaria and me made some tests and this is the better way to display the item)

File: src/list.tsx (hunk 4 above)
Comment (sergivillar) [reply src/list.tsx:485]: needed to not use the default render, that will add the -2 hack that you see above, to let the list component works as it was previously

**Review by atabel — APPROVED** [review]

File: src/radio-button.tsx (hunk 2 above)
Comment (pladaria) [reply src/radio-button.tsx:147]: Not break but it will probably displace the content in unexpected ways

**Review by pladaria — APPROVED** [review]
